profiling/Cargo.toml

```
@@ -6,16 +6,18 @@ license = "Apache-2.0"
 rust-version = "1.64"
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
```

Is rlib okay here? Doesn't it break LTO? Or did we fix that by using newer CLI invocations to specify crate-types during the build in CI?

---

Nope, that was a left over from me locally running the benchmarks
```suggestion
crate-type = ["cdylib"]
```